.prowci/config.yaml

```diff
@@ -1,7 +1,7 @@
 log_level: debug
 
 tide:
-  # target_url: http://ci-bot-aks-ingress.eastus.cloudapp.azure.com/tide.html
+  # target_url: http://prow-ci-bot-ingress.eastus.cloudapp.azure.com/tide.html
```

Comment:
Is the new prow CI bot ready for acs-engine (not just aks-engine)?

Reply:
no this bot runs for aks-engine, not acs-engine

Reply:
I fixed the setup to make it work automatically so we do want these changes though